src/core/composer/qgscomposerpointsbasedshape.h

```diff
@@ -0,0 +1,155 @@
+/***************************************************************************
+                         qgscomposerpointsbasedshape.h
+    begin                : March 2016
+    copyright            : (C) 2016 Paul Blottiere, Oslandia
+    email                : paul dot blottiere at oslandia dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSCOMPOSERPOINTSBASEDSHAPE_H
+#define QGSCOMPOSERPOINTSBASEDSHAPE_H
+
+#include "qgscomposeritem.h"
+#include <QBrush>
+#include <QPen>
+
+/** An abstract composer item that provides generic methods for points based
+ * shapes such as polygon or polylines. */
```

Comment:
Can you add a:

    @note added in QGIS 2.16 

here too (and to other new classes/methods). It helps PyQGIS users to know what classes and methods they can use in their plugins.